packages/regl-worldview/src/camera/CameraListener.js

```diff
@@ -19,18 +19,47 @@ const KEYBOARD_MOVE_SPEED = 0.3;
 const KEYBOARD_ZOOM_SPEED = 150;
 const KEYBOARD_SPIN_SPEED = 1.5;
 
+const ACTIONS = {
+  MOVE_DOWN: 'moveDown',
+  MOVE_LEFT: 'moveLeft',
+  MOVE_RIGHT: 'moveRight',
+  MOVE_UP: 'moveUp',
+  ROTATE_LEFT: 'rotateLeft',
+  ROTATE_RIGHT: 'rotateRight',
+  TILT_DOWN: 'tiltDown',
+  TILT_UP: 'tiltUp',
+  ZOOM_IN: 'zoomIn',
+  ZOOM_OUT: 'zoomOut',
+};
```

Comment:
You're right. Since this is a shared component, I was just being extremely careful to prevent making any breaking changes but this is overboard. I'd prefer to use Object.freeze as follows: ```Object.freeze({
tiltUp: 'tiltUp',
})```